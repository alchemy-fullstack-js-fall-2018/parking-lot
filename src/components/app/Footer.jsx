import React from 'react';
import styled from 'styled-components';
import { config, colors } from '../../config';

const StyledFooter = styled.footer`
  background-color: ${colors.primary};
  p {
      color: ${colors.secondary};
  }
`;

const Footer = () => {
  return (
    <StyledFooter>
      <p>{config.footer}</p>
    </StyledFooter>
  );
};

export default Footer;
