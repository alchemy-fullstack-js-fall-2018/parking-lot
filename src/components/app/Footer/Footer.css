@import '../../../styles/variables.css';

.footer {
  background-color: var(--primary);
  p {
      color: var(--secondary);
  }
}
