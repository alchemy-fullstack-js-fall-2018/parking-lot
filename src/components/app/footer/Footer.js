import React, { Component } from 'react';

import styles from './Footer.css';

export default class Footer extends Component {


  render() {

    return (
      <footer className={styles.footer}>
        <p>Made with the utmost seriousness - 2018</p>
      </footer>
    );
  }


}
