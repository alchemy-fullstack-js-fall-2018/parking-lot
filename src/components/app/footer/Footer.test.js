import React from 'react';
import Footer from './Footer';
import { shallow } from 'enzyme';

describe('Footer Component', () => {
  it('should be a footer', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
