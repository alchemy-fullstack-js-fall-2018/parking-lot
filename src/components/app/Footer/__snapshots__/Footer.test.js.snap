// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders 1`] = `
<footer
  className="footer"
>
  <p>
    Made with the utmost seriousness - 2018
  </p>
</footer>
`;
